fe/fe-core/src/main/java/org/apache/doris/statistics/PartitionStatisticBuilder.java

```diff
@@ -0,0 +1,160 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.statistics;
+
+import org.apache.doris.analysis.LiteralExpr;
+import org.apache.doris.statistics.util.Hll128;
+
+public class PartitionStatisticBuilder {
+    private double count;
+    private Hll128 ndv;
+    private double avgSizeByte;
+    private double numNulls;
+    private double dataSize;
+    private double minValue = Double.NEGATIVE_INFINITY;
+    private double maxValue = Double.POSITIVE_INFINITY;
```

Comment:
why these two have default value?

Reply:
The default value of double type is 0.0 which may cause generate 0 row count estimation. Set the default value to NEGATIVE_INFINITY and POSITIVE_INFINITY to cover this case.